packages/table/src/react/hooks/useColorDropdownMenu.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export const useColorDropdownMenuState = ({
```

Comment:
We stop packaging such hooks going forward, this can be moved into the component